examples/sglang/components/worker.py

```
@@ -80,6 +87,23 @@ async def async_init(self):
+                kv_metrics = await self.receive_metrics_from_scheduler.recv_pyobj()
+                self.metrics_publisher.publish(
+                kv_metrics.request_active_slots,
+                kv_metrics.request_total_slots,
+                kv_metrics.kv_active_blocks,
+                kv_metrics.kv_total_blocks,
+                kv_metrics.num_requests_waiting,
+                kv_metrics.gpu_cache_usage_perc,
+                kv_metrics.gpu_prefix_cache_hit_rate,
+                kv_metrics.data_parallel_rank)
+
+            except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)